src/App/Components/DisplayArticle/readArticle.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable max-len */
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import propTypes from 'prop-types';
+import Popular from '../Popular/popular';
+import ReadArticleActions from '../../../Redux/Actions/readArticleActions';
+import PopularArticleAction from '../../../Redux/Actions/readPopularActions';
+import ArticleNotFound from '../ArticleNotFound/ArticleNotFound';
+import BreadCrumb from './breadCrumb';
+import ArticleBody from './articleBody';
+import Loader from '../Loader/Loader';
+
+const { readArticle, getTags } = ReadArticleActions;
+const { readPopularArticle } = PopularArticleAction;
+
+export class ReadArticle extends Component {
+  state = { tags: [], isLoading: true, articles: [{ title: '', User: {}, Category: {} }] }
+
+  componentWillMount() {
```

Comment:
@MCFrank16 https://github.com/styled-components/styled-components/issues/1575

use componentDidMount instead of componentWillMount